### PR TITLE
SAK-44538 postem Improve responsiveness of add screen

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-44538

### CODE REVIEW

File: postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html [hunk 1]
```diff
@@ -64,11 +64,11 @@ <h1 th:text="#{create_update}" /></h1>
...
                 </div>
-                <div class="col-sm-4">
-                    <input type="checkbox" name="gradeBookFeedBackAvail" id="released"
-                        th:field="*{released}" /> <label for="gradeBookFeedBackAvail"
-                        class="form-control-label" th:text="#{release}">Release
-                        feedback to participants?</label>
+                <div class="col-sm-4 text-nowrap">
+                    <input type="checkbox" name="gradeBookFeedBackAvail" id="released" th:field="*{released}" />
+                    <label for="released" class="form-control-label" th:text="#{release}">
+                        Release feedback to participants?
```
Comment (jesusmmp) [inline postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html:70]: i18n?

Comment (adrianfish) [reply postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html:70]: Not needed. Thymeleaf replace the tag contents. It's just a placeholder.

Comment (adrianfish) [reply postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html:70]: th:text does the magic.

Comment (jesusmmp) [reply postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html:70]: Great!. Thanks

**Pull request merged**